aten/src/ATen/native/cpu/ScatterGatherKernel.cpp

```diff
@@ -685,7 +690,8 @@ std::pair<K*, V*> radix_sort_parallel(
 //
 
 template <typename scalar_t, ReductionType reduce>
-void cpu_scatter_reduce_expanded_index(const Tensor& self, const Tensor& index, const Tensor& src, bool include_self) {
+void cpu_scatter_reduce_expanded_index(const Tensor& self, const Tensor& _index, const Tensor& src, bool include_self) {
+  Tensor index = _index.to(ScalarType::Long);
```

Comment:
ooc, we are not changing supported types for scatter yet, right?

Reply:
Oops. I removed it.